src/Tests.IntegrationTests/appsettings.json

```diff
@@ -1,6 +1,14 @@
 {
   "Bot1Url": "http://localhost:9012",
-  "Bot1ApiKey":  "testing",
+  "Bot1ApiKey": "testing",
   "CentOpsUrl": "http://localhost:9014",
-  "CentOpsApiKey": "testingadmin"
+  "CentOpsApiKey": "testingadmin",
+  "ClassifierInternalUrl": "http://classifier",
+  "DmrInternalUrl": "http://dmr",
+  "Bot1InternalUrl": "http://bot1",
+  "UseFixture": "true"
```

Comment:
We agreed on a call to change `UseFixture` to `GenerateData`/`GenerateEntities` as the current name still makes us think about exactly what is happening if we set this property to `true`/`false`.

Reply:
Have gone for `GenerateTestData`